docker-compose.yml

```diff
@@ -1,12 +1,42 @@
 version: "3.9"
 services:
-  arena-client:
+  sc2_controller:
+    image: aiarena/arenaclient-sc2:v0.4.3
+    environment:
+      - "ACSC2_PORT=8083"
+      - "ACSC2_PROXY_HOST=proxy_controller"
     volumes:
-      - ./config.py:/root/aiarena-client/config.py
-      - ./:/root/aiarena-client/host
       # SC2 Maps Path
       # Set this as "- PATH_TO_YOUR_MAPS_FOLDER:/root/StarCraftII/maps"
-      - C:\Program Files (x86)\StarCraft II\Maps:/root/StarCraftII/maps  # Standard windows SC2 maps path
-#      - ./maps:/root/StarCraftII/maps                                   # Local maps folder
+#      - C:\Program Files (x86)\StarCraft II\Maps:/root/StarCraftII/maps # Standard windows SC2 maps path
+      - ./maps:/root/StarCraftII/maps                                   # Local maps folder
 #      - ~/StarCraftII/maps:/root/StarCraftII/maps                       # Relatively standard linux SC2 maps path
-    image: aiarena/arenaclient
\ No newline at end of file
+
+  bot_controller1:
+    image: aiarena/arenaclient-bot:v0.4.3
+    volumes:
+      - "./bots:/bots"
```

Comment:
Should a `logs` volume be mounted here? (and for bot2 controller)

Reply:
I tried the log mapping and removed it because the only logs I could see there were also present in the docker container output.

I figured it's a simpler setup, not having the logs folder and instead looking at the container output.

Reply:
Apologies @raspersc2 - I've just realized you're talking about the bot logs, not the arena client logs.
I will look into putting this in.